source/scss/stinson.scss

```diff
@@ -0,0 +1,37 @@
+/*
+ *
+ * Stinson
+ *
+ */
+
+@import 'shared';
+
+// mixin to extend stinson filter
+// @mixin stinson
+// @param $filters... {filter} - Zero to many css filters to be added
+// @example
+//   img {
+//     @include stinson;
+//   }
+//   or
+//   img {
+//     @include stinson(blur(2px));
+//   }
+//   or
+//   img {
+//     @include stinson(blur(2px)) {
+//       /*...*/
+//     };
+//   }
+@mixin stinson($filters...) {
+  @include filter-base;
+  filter: contrast(0.75) saturate(0.85) $filters;
```

Comment:
I would add a bit of brightness too, as its looking a bit darker than the filer, maybe `brightness(1.15)`